src/session.rs

```diff
@@ -409,6 +413,10 @@ impl Session {
             self.twcc_rx_register.update_seq(extended.into(), now);
         }
 
+        // Store largest seen seq_no for the SSRC. This is used in case we get SSRC changes
+        // like A -> B -> A. When we go back to A, we must keep the ROC.
```

Comment:
I don't fully understand this. The ROC is per SSRC so why should it matter if two separate SSRCs have different values, outside of RTX repaired packets I don't think these should be combined

Reply:
It's not about combining. I'll explain in PR description.